Makefile

```diff
@@ -64,7 +64,7 @@ container-image:
 		--build-arg HUGO_VERSION=$(HUGO_VERSION)
 
 container-build: module-check
-	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo
+	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --minify
```

Comment:
This should be double checked, minify was causing problems for svgs :(

It was fixed in the upstream [minify library](https://github.com/tdewolff/minify) as of 2.7.4, but to my knowledge has not been included in Hugo yet. I had to pull it out of the contrib site config  for breaking the kubernetes logo.
ref: https://github.com/kubernetes-sigs/contributor-site/pull/131
ref: https://github.com/gohugoio/hugo/issues/6892